app/assets/javascripts/app/discussions/templates/edit_header_body.html.slim

```diff
@@ -0,0 +1,6 @@
+md-button.md-primary.md-hue-1 ui-sref="discussions.new.title"
+  i.fa.fa-chevron-left.u-alpha
```

Comment:
Exactly, it's a different button, and it's used 4 times for now. The utilities are two avoid and for things that are not repeated anywhere and you're sure will not.